cves/kernel/CVE-2017-17805.yml

```diff
@@ -452,7 +504,14 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    I believe that the main mistake made that caused this vulnerability was
+    a lack of a zero trust policy. The vulnerability resulted from a lack of
```

Comment:
I would include a brief explanation of what a zero trust policy is.

Reply:
Good call!